app/models/mixins/reportable_mixin.rb

```diff
@@ -11,6 +11,10 @@ def sortable?
       true
     end
 
+    def scope_by_tenant?
+      false
```

Comment:
In what way?  It seemed like we wanted to define who would be scoped by tenant so I thought the class should have that answer.  What were you thinking?